helm/reana/templates/cronjobs.yaml

```diff
@@ -257,3 +257,82 @@ spec:
           - name: reana-code
             hostPath:
               path: /code/reana-server
+---
+{{- if not (contains "forever" (.Values.interactive_sessions.maximum_duration | quote)) }}
```

Comment:
We are repeating the definition of cronjobs many times, resulting in a lot of copy-paste. Should we investigate whether we can reduce duplication, perhaps by defining new templates?

We can do this in another issue/PR.

Reply:
Created an issue for this https://github.com/reanahub/reana/issues/695